Source/Parallelization/WarpXRegrid.cpp

```diff
@@ -166,6 +166,69 @@ WarpX::RemakeLevel (int lev, Real /*time*/, const BoxArray& ba, const Distributi
             rho_fp[lev] = std::move(pmf);
         }
 
+#ifdef WARPX_USE_PSATD
+        if (WarpX::maxwell_solver_id == MaxwellSolverAlgo::PSATD) {
```

Comment:
Are the "WarpX::" prefixes needed here since this is part of the WarpX class?

Reply:
Maybe not needed, I have now removed these.